client/src/components/BtnGoBack.jsx

```diff
@@ -9,7 +9,7 @@ function BtnGoBack() {
   };
   return (
     <button
-      aria-labelledby="button-back"
+      aria-label="Bouton retour"
```

Comment:
"Retour vers la page précédente" would have been better, as I said on your PR about aria-label

Reply:
Yes I have done this one before getting your info, so I will fix it.